scripts/legacy/scil_screenshot_bundle.py

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
```

Comment:
If everyone is at the lab, sometimes, let's discuss the renaming choices together!